src/main/java/com/ericsson/ei/frontend/utils/RegExProviderUtils.java

```diff
@@ -0,0 +1,24 @@
+package com.ericsson.ei.frontend.utils;
+
+import java.io.IOException;
+
+import org.json.JSONException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.ericsson.eiffelcommons.utils.RegExProvider;
+
+public class RegExProviderUtils {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RegExProviderUtils.class);
+
+    public static String getRegEx(String key) {
```

Comment:
It feels unnecessary to do logic regarding the retrieval of the regex here. The base lib should in that case return it as string if that is the wanted structure. Could then skip this method and where you want to use the regex you could do for example RegExProvider.getEmailRegEx() which will return a string directly. This will need to implement that function in the shared lib though.

Reply:
New pull request implements functionality as per our team discussion